opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx

```diff
@@ -0,0 +1,142 @@
+import { ExclusionListEdition_edition$key } from '@components/settings/exclusion_lists/__generated__/ExclusionListEdition_edition.graphql';
+import { Option } from '@components/common/form/ReferenceField';
+import { ExclusionListEntityTypes } from '@components/settings/exclusion_lists/__generated__/ExclusionListsCreationFileAddMutation.graphql';
+import AutocompleteField from '../../../../components/AutocompleteField';
+import { fieldSpacingContainerStyle } from '../../../../utils/field';
+import useApiMutation from '../../../../utils/hooks/useApiMutation';
+import MarkdownField from '../../../../components/fields/MarkdownField';
+import TextField from '../../../../components/TextField';
+import { useFormatter } from '../../../../components/i18n';
+
+export const exclusionListMutationFieldPatch = graphql`
+  mutation ExclusionListEditionFieldPatchMutation($id: ID!, $input: [EditInput!]!) {
+    exclusionListFieldPatch(id: $id, input: $input) {
+      ...ExclusionListsLine_node
+      ...ExclusionListEdition_edition
+    }
+  }
+`;
+
+const exclusionListEditionFragment = graphql`
+  fragment ExclusionListEdition_edition on ExclusionList {
+    id
+    name
+    description
+    exclusion_list_entity_types
+  }
+`;
+
+export const exclusionListEditionQuery = graphql`
+  query ExclusionListEditionQuery($id: String!) {
+    exclusionList(id: $id) {
+      ...ExclusionListEdition_edition
+    }
+  }
+`;
+
+const exclusionListValidation = (t: (n: string) => string) => Yup.object().shape({
+  name: Yup.string().required(t('This field is required')),
+  description: Yup.string().nullable(),
+});
+
+interface ExclusionListEditionComponentProps {
+  queryRef: PreloadedQuery<ExclusionListEditionQuery>;
+  onClose: () => void;
+}
+
+interface ExclusionListEditionValues {
+  name: string;
+  description?: string | null;
+  exclusion_list_entity_types: Option[];
+}
+
+const ExclusionListEdition: FunctionComponent<ExclusionListEditionComponentProps> = ({
+  queryRef,
+  onClose,
+}) => {
+  const { t_i18n } = useFormatter();
+  const { exclusionList } = usePreloadedQuery<ExclusionListEditionQuery>(exclusionListEditionQuery, queryRef);
+  const data = useFragment<ExclusionListEdition_edition$key>(exclusionListEditionFragment, exclusionList);
+  const [commitFieldPatch] = useApiMutation(exclusionListMutationFieldPatch);
+  const onSubmit = (name: string, value: string[]) => {
+    // TODO : Use useFormEditor ?
+    commitFieldPatch({
+      variables: {
+        id: data?.id,
+        input: [{ key: name, value }],
+      },
+    });
+  };
+
+  const initialValues: ExclusionListEditionValues = {
+    name: data?.name ?? '',
+    description: data?.description,
+    exclusion_list_entity_types: (data?.exclusion_list_entity_types ?? []).map((type) => ({
+      value: type,
+      label: type,
+    })),
+  };
+
+  const entityTypes: ExclusionListEntityTypes[] = ['IPV4_ADDR', 'IPV6_ADDR', 'DOMAIN_NAME', 'URL'];
```

Comment:
we should list all possible stix cyber observable entity types, not this static list, which doesn't even correspond to a valid entity type

Reply:
you should be able to get them like this : 
```
const { schema: { scos: entityTypes } } = useSchema();
```

Reply:
Done. We need to verify with the product about this list, and if it's really able to create exclusion list for all observable types. Valentin is on it.